.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Tests
+on: [push, pull_request, create]
+jobs:
+  runTests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+      - name: Install dependencies
+        run: sudo apt-get install -y libevent-dev python3-pip python3-virtualenv
+      - name: Run Tests Tests
+        run: |
+          mvn package
+          ./src/test/resources/run-s3-tests.sh
+      - name: Build Docker Image
+        run: ./publish_img.sh
+        env:
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+          DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
```

Comment:
Make sure to add these two secrets before merging

Reply:
I added secrets.  Let's merge this then try to force a build.

Reply:
Could you take a look at https://github.com/gaul/s3proxy/runs/4211919696?check_suite_focus=true#step:6:18402 ?  I suspect that `REPO` needs to be set.